server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/BalancedShardsAllocator.java

```diff
@@ -141,6 +143,24 @@ public ShardAllocationDecision decideShardAllocation(final ShardRouting shard, f
         return new ShardAllocationDecision(allocateUnassignedDecision, moveDecision);
     }
 
+    private void failedAllocationOfNewPrimaries(RoutingAllocation allocation){
```

Comment:
Minor change:

```suggestion
    private void failAllocationOfNewPrimaries(RoutingAllocation allocation) {
```

(change name to the imperative mood, and fix whitespace)